### PR TITLE
[T4A6][W13-B1] Lim Hui Qi

### PR DESCRIPTION
**Pull request by LuMiN0uSaRc**
Use JUnit to write automated tests

### CODE REVIEW

**Comment by LuMiN0uSaRc** [comment]
Ready to Review

File: src/seedu/addressbook/ui/UtilsTest.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.*;
```
Comment (okkhoy) [inline src/seedu/addressbook/ui/UtilsTest.java:3]: don't import everything. import only what is necessary.

File: src/seedu/addressbook/ui/UtilsTest.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+import java.util.ArrayList;
+import org.junit.Test;
+
+public class UtilsTest {
+
+	@Test
+	public void isAnyNull_noNull() {
```
Comment (okkhoy) [inline src/seedu/addressbook/ui/UtilsTest.java:10]: the test name should have 3 parts `<method being tested>_<what is being tested>_<what is the expected return value>`
you can eliminate part 3 and 2 only if you have broadened the scope of the test to include testing multiple things.

File: src/seedu/addressbook/ui/UtilsTest.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+	public void isAnyNull_hasNull() {
+		Object[] testObjects = new Object[] {null, new Integer(123)};
+		boolean result = Utils.isAnyNull(testObjects);
+		assertEquals(result, true);
+	}
+	
+	@Test
+	public void elementAreUnique_isUnique() {
+		ArrayList<Object> testObjects = new ArrayList<Object>();
+		testObjects.add(new String("12313"));
```
Comment (okkhoy) [inline src/seedu/addressbook/ui/UtilsTest.java:26]: going further you can consider moving all the steps necessary for the test to run (or preconditions) to a set up method before beginning the tests; correspondingly can put all the tear down in a method after the tests.

**Comment by okkhoy** [comment]
@LuMiN0uSaRc sorry to harp on the same thing, but please submit clean PRs for the next tutorial!
Some comments added. Please ack & close the PR after reading comments.

**Comment by LuMiN0uSaRc** [comment]
Noted with thanks!

**Pull request closed without merging**